roles/openshift_health_checker/openshift_checks/docker_image_availability.py

```diff
@@ -153,7 +154,15 @@ def is_image_local(self, image):
 
     def known_docker_registries(self):
         """Build a list of docker registries available according to inventory vars."""
-        regs = list(self.get_var("openshift_docker_additional_registries", default=[]))
+        regs = self.get_var("openshift_docker_additional_registries", default=[])
+        # https://bugzilla.redhat.com/show_bug.cgi?id=1497274
+        # if the result was a string type, place it into a list. We must do this
+        # as using list() on a string will split the string into it's characters.
```

Comment:
"its" :)

Not worth holding up the PR over.

Reply:
I'll fix it anyway 😆